project/Dependencies.scala

```diff
@@ -24,6 +24,7 @@ object Dependencies {
     val Scalacheck = "1.14.0"
     val Mockito = "1.15.0"
     val GCNio = "0.121.2"
+    val Elastic4s = "7.9.1"
```

Comment:
I think we could write some nice documentation already! 
Se we would just need to create a new `elasticsearch.md` under `./docs` folder. 
And then we will have to do minimal changes (which you can take other connectors as reference) on `website/sidebars.json`, `website/pages/en/index.js` and `website/i18/en.json`. 

To see if the changes were correctly applied:
```
sbt docs/docusaurusCreateSite
cd website/
yarn start
```

Reply:
Just before writing documentation, I wanted to let you know that some of the existing apis are getting moved to a more purely functional design, using _cats effect_ to acquire and release resources (in this case it would be `ElasticClient`).

Maybe we can go with this implementation as a first version of the elasticsearch connector, and make those changes if we see it convenient in the future. I leave that up to you.

If you are interested in doing it now, you can follow the this PR as an example: https://github.com/monix/monix-connect/pull/379

Reply:
OK. I'll try this next commit.

Reply:
Cool!